build.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
 }
 
 group 'thm.mope.lsp'
-version '0.1.0'
+version '0.0.9-test3'
```

Comment:
Just a small note, but be careful with *downgrading* version numbers. This will confuse any package manager, because the assumptions with version numbers is always that higher versions are more up to date than lower ones. Some package managers therefore only allow you to publish versions of your software that are a *direct* upgrade from an already existing version. So if you have, for example `0.0.9` and `0.1.0`, you can upload versions `0.0.10`, `0.1.1`, `0.2.0`, or `1.0.0`, but these are your only options. There is no harm in already sticking to this scheme during initial development.